### PR TITLE
Changed status to deleted

### PR DESCRIPTION
**Pull request by deekshacheruku**
The commit is found using git bisect. All the logs are in my VM ```deeksha5@fa23-cs527-117``` and in the ```path /home/deeksha5/ignite-3/deleted-test```

ConfigurationAsmGeneratorTest is renamed as ConfigurationTreeGeneratorTest in the commit ```https://github.com/apache/ignite-3/commit/b48ddcba7cd2bd3b9a053ae131c25b44a0400e27```

testConstructInternalConfig has been renamed with testConstructExtendedConfig and has a different method body => Update status as ```Deleted``` with the commit ```https://github.com/apache/ignite-3/commit/960ce0a72dc8486bca6458f82471dc6d35265482```

I will make another PR for this test and the tests in https://github.com/TestingResearchIllinois/idoft/pull/1037 if they are flaky in gradle. I feel it is too much for this PR to have both the deletion and addition. @darko-marinov Let me know if there are any concerns. Thanks in advance!

This is small PR with respect to https://github.com/TestingResearchIllinois/idoft/pull/1037

### CODE REVIEW

**Comment by darko-marinov** [comment]
`git bisect` is an overkill to find that a test is deleted. Was the test class deleted (how do you know it was not renamed or moved)? Or was the test method deleted (again, how did you check if the logic was not moved elsewhere)?

**Comment by deekshacheruku** [comment]
The test class ConfigurationAsmGeneratorTest is renamed as ConfigurationTreeGeneratorTest in the commit https://github.com/apache/ignite-3/commit/b48ddcba7cd2bd3b9a053ae131c25b44a0400e27 but then in https://github.com/apache/ignite-3/commit/960ce0a72dc8486bca6458f82471dc6d35265482 the test method was renamed and there were few lines that were added (method body changed) @darko-marinov

**Comment by darko-marinov** [comment]
Can you put a direct link to the hunk for the relevant change, rather than a link to the entire commit, which has changed hundreds of lines? You can even add two changes (for more "points"), one for `MovedOrRenamed` for `b48ddcba` and the other for `Deleted` for `960ce0a7`.

**Comment by deekshacheruku** [comment]
Link For testclassRename: https://github.com/apache/ignite-3/commit/b48ddcba7cd2bd3b9a053ae131c25b44a0400e27#diff-db303dade1fcce8c4f4ce2832bf91c33db3ed3978fb447ad581aeb8cf1559502

Link for testMethodRename: https://github.com/apache/ignite-3/commit/960ce0a72dc8486bca6458f82471dc6d35265482#diff-db303dade1fcce8c4f4ce2832bf91c33db3ed3978fb447ad581aeb8cf1559502L252-R280

@darko-marinov

**Comment by darko-marinov** [comment]
Great! Those two links look convincing. You should add both lines into `pr-data.csv`. But isn't even the latter method rename largely preserving the test logic? Ideally you'd check if the test remains `ID` at that commit.

**Comment by deekshacheruku** [comment]
Done @darko-marinov I have added both the lines. Let me know if there is anything else. Thanks a lot!

File: pr-data.csv [hunk 1]
```diff
@@ -2030,7 +2030,8 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,or
...
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/cli,org.apache.ignite.cli.deprecated.IgniteCliInterfaceTest$Cluster.initSuccess,ID,Accepted,https://github.com/apache/ignite-3/pull/888,The PR was closed but the commit https://github.com/apache/ignite-3/commit/81851d38f6197ea414552148d9db372c09b43eb8 was added to the main branch of the repository
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.asm.ConfigurationAsmGeneratorTest.testConstructInternalConfig,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.asm.ConfigurationAsmGeneratorTest.testConstructInternalConfig,ID,Deleted,,https://github.com/apache/ignite-3/commit/960ce0a72dc8486bca6458f82471dc6d35265482
```
Comment (darko-marinov) [inline pr-data.csv:2033]: Shouldn't this have a different class name? Also, is the test really being `Deleted` or just renamed? Could you run NonDex in `960ce0a72dc8486bca6458f82471dc6d35265482` to check if the test is still `ID`?

Comment (deekshacheruku) [reply pr-data.csv:2033]: I have added the correct class name. I can add the test in gradle file if it is flaky in the next PR. @darko-marinov

**Comment by darko-marinov** [comment]
Thanks for all the updates. We can accept this latest version. Enough learning has been demonstrated. To some degree, I don't care that the dataset be 100% accurate for the test evolution. We care the most about the latest versions.

**Comment by deekshacheruku** [comment]
Thanks a lot! @darko-marinov

**Pull request merged**